environments/dev/dom1-infra-int.yml

```
@@ -0,0 +1,24 @@
+name: "dom1.infra.int"
+vnet_links:
+  - link_name: "core-infra-vnet-mgmt"
+    vnet_id: "/subscriptions/ed302caf-ec27-4c64-a05e-85731c3ce90e/resourceGroups/rg-mgmt/providers/Microsoft.Network/virtualNetworks/core-infra-vnet-mgmt"
+  - link_name: "bastion-nonprod-vnet"
+    vnet_id: "/subscriptions/b44eb479-9ae2-42e7-9c63-f3c599719b6f/resourceGroups/bastion-nonprod-rg/providers/Microsoft.Network/virtualNetworks/bastion-nonprod-vnet"
+  - link_name: "reform-mgmt-core-vnet"
+    vnet_id: "/subscriptions/ed302caf-ec27-4c64-a05e-85731c3ce90e/resourceGroups/reformMgmtCoreRG/providers/Microsoft.Network/virtualNetworks/reformMgmtCoreVNet"
+A:
+  - name: juror-api
+    record:
+    - 10.100.197.186
+    ttl: 300
+  - name: juror-api-nle
+    record:
+    - 10.72.1.211
+    ttl: 300
+cname:
+  - name: juror-api
+    record: juror-api.dom1.infra.int
+    ttl: 300
+  - name: juror-api-nle
+    record: juror-api-nle.dom1.infra.int
+    ttl: 300   
```

this isn't doing anything as far as i cann tell?
```suggestion
cname: []
```